pkg/kubectl/cmd/get.go

```diff
@@ -543,7 +543,7 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 func addOpenAPIPrintColumnFlags(cmd *cobra.Command) {
 	cmd.Flags().Bool(useOpenAPIPrintColumnFlagLabel, false, "If true, use x-kubernetes-print-column metadata (if present) from openapi schema for displaying a resource.")
 	// marking it deprecated so that it is hidden from usage/help text.
-	cmd.Flags().MarkDeprecated(useOpenAPIPrintColumnFlagLabel, "its an experimental feature.")
+	cmd.Flags().MarkDeprecated(useOpenAPIPrintColumnFlagLabel, "Its an experimental feature.")
```

Comment:
Rebased and added change as suggested. May I ask a git question. I was unable to push the changes to GitHub because 'Updates were rejected because the tip of your current branch is behind'. I used `git push -f` to push the changes. Is this acceptable practice or is there a better way?